README.md

```diff
@@ -1,4 +1,57 @@
-# Safe Relay Service
+<div align="center">
+	<img width="80" src="https://raw.githubusercontent.com/CirclesUBI/.github/main/assets/logo.svg" />
+</div>
+
+<h1 align="center">safe-relayer-service</h1>
+
+<div align="center">
+ <strong>
+   Relay Tx Service for Gnosis Safe
+ </strong>
+</div>
+
+<br />
+
+<div align="center">
+  <!-- Licence -->
+  <a href="https://github.com/CirclesUBI/circles-core/blob/main/LICENSE">
+    <img src="https://img.shields.io/github/license/CirclesUBI/circles-core?style=flat-square&color=%23cc1e66" alt="License" height="18">
+  </a>
+  <!-- CI status -->
+  <a href="https://github.com/CirclesUBI/circles-core/actions/workflows/tests.yml">
+   <img src="https://img.shields.io/github/workflow/status/CirclesUBI/safe-relay-service/Build%20docker%20and%20upload%20to%20registry?color=%2347cccb&label=test&style=flat-square " alt="CI Status" height="18">
+  </a>
```

Comment:
This still addresses the circles-core

Reply:
shouldnt have done this in a hurry :sweat_smile: